ui-components/src/headers/PageHeader.tsx

```diff
@@ -27,9 +39,24 @@ const PageHeader = (props: PageHeaderProps) => {
       <hgroup className="page-header__group">
         <h1 className="page-header__title">{props.title}</h1>
         {props.subtitle && <p className="page-header__lead"> {props.subtitle}</p>}
-      </hgroup>
+        {props.children}
 
-      {props.children}
+        {!!props?.tabNav?.length && (
+          <TabNav className="page-header__secondary-nav">
+            {props.tabNav?.map((tab) => (
+              <Tab
```

Comment:
FYI, this was renamed to `TabNavItem` in #1305…also might have a conflict with your type above.

Reply:
Fixed, thanks.